docs/common/agent-builder/_adding-nodes.mdx

```diff
@@ -0,0 +1,19 @@
+Whenever you create a fresh Agent Builder project for your AI, we'll start you out with a root node and a sequence node to form the basis of your tree.
```

Comment:
I think this can be a bit more concise

```suggestion
New Agent Builder projects start with a root node and a sequence node to form the basis of your tree.
```

Reply:
Updated